src/shapes/IText/ITextKeyBehavior.test.ts

```diff
@@ -0,0 +1,348 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
Many tests here are the same apart from params and can become 
test.each

Reply:
no i really like the tests one after the other is on level of nesting less, they are never exactly the same.
You end up having parameters like selectionStart, selectionEnd, direction, expected value and you end up writing the test in the parameters instead of the test function.